string type
int32 state

# type="nav_action"
inhus/NavAction nav_action

# type="wait_action"
inhus/WaitAction wait_action
